doc/source/getting_started/index.rst

```diff
@@ -9,82 +9,49 @@ Getting started
 Installation
 ------------
 
-.. raw:: html
+.. panels::
+    :card: + install-card
+    :column: col-lg-6 col-md-6 col-sm-12 col-xs-12 p-3
 
-    <div class="container">
-        <div class="row">
-            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
-                <div class="card install-card shadow w-100">
-                <div class="card-header">
-                    Working with conda?
-                </div>
-                <div class="card-body">
-                    <p class="card-text">
+    Working with conda?
+    ^^^^^^^^^^^^^^^^^^^
 
-pandas is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__ distribution and can be
-installed with Anaconda or Miniconda:
+    pandas is part of the `Anaconda <https://docs.continuum.io/anaconda/>`__
+    distribution and can be installed with Anaconda or Miniconda:
 
-.. raw:: html
+    ++++++++++++++++++++++
 
-                    </p>
-                </div>
-                <div class="card-footer text-muted">
+    .. code-block:: bash
 
-.. code-block:: bash
+        conda install pandas
 
-   conda install pandas
+    ---
 
-.. raw:: html
+    Prefer pip?
+    ^^^^^^^^^^^
 
-                </div>
-                </div>
-            </div>
-            <div class="col-lg-6 col-md-6 col-sm-12 col-xs-12 d-flex install-block">
-                <div class="card install-card shadow w-100">
-                <div class="card-header">
-                    Prefer pip?
-                </div>
-                <div class="card-body">
-                    <p class="card-text">
+    pandas can be installed via pip from `PyPI <https://pypi.org/project/pandas>`__.
 
-pandas can be installed via pip from `PyPI <https://pypi.org/project/pandas>`__.
+    ++++
 
-.. raw:: html
+    .. code-block:: bash
 
-                    </p>
-                </div>
-                <div class="card-footer text-muted">
+        conda install pandas
```

Comment:
This should be
```suggestion
        pip install pandas
```

Reply:
Whoops, copy/paste error ;) Thanks for the catch!